modules/home/assets/js/icons/youtube-icon.js

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { SvgIcon, styled } from '@elementor/ui';
+
+const YoutubeIcon = ( props ) => {
```

Comment:
I see that no props are transferred to this component.

Reply:
@davseve I think this is the default way in which we do this, in case `props` would transferred.